.docker/Dockerfile-build

```diff
@@ -13,13 +14,15 @@ ENV CGO_ENABLED 1
 
 RUN go mod download
 
+FROM base as builder
```

Comment:
We are using the same image as in the previous step, do we need to or can we leave this out?

Reply:
it will still work porperly without this